Readme.md

```diff
@@ -100,6 +101,32 @@ describe(...
 )
 ```
 
+## Analyze all logs
+
+In some circumstances Jest is infamouse for logging because it overwrites the bash output. If you're experiencing this kind of logging frustration and you need to analyze all the `console.log` call arguments (tipically to understand why a test is failing) you can temporary add an expect statement on the console itself.
```

Comment:
Just a few spelling errors. 
"infamouse" should be "infamous"
"tipically" should be "typically"
"temporary" should be "temporarily"

Reply:
Oh sorry, I've just fixed them!